lib/ads/interstitial_ad_manager.js

```diff
@@ -214,41 +214,52 @@ shaka.ads.InterstitialAdManager = class {
       return;
     }
     this.updatePlayerConfig_();
-    let cuepointsChanged = false;
     const interstitialsAd = await this.getInterstitialsInfo_(interstitial);
     if (interstitialsAd.length) {
       for (const interstitialAd of interstitialsAd) {
```

Comment:
I hadn't noticed this before, but `interstitialsAd` is a really rough variable name. It's hard to tell from `interstitialAd` at a glance, since the added "s" is in the middle of the variable name.
I get that it's one ad but multiple interstitials, so `interstitialAds` wouldn't be right...
But couldn't we call the variables `adInterstitials` and `adInterstitial`? Then at least the "s" gets to go on the end of the variable.

Reply:
Changed!